.github/workflows/release_staging.yml

```diff
@@ -33,33 +35,8 @@ jobs:
         RUDDERSTACK_URL: ${{ vars.RUDDERSTACK_URL }}
     - name: Run tests
       run: npm test
-    - name: Versioning
-      uses: "./.github/actions/versioning"
-      with:
-        version_name: staging
     - name: Publish to Cloudflare Pages Staging 
       uses: "./.github/actions/publish_to_pages_staging"
       with:       
-        version: $(echo cat packages/core/dist/version | bash)
         CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
         CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
-
-
-  send_slack_notification:
```

Comment:
Based on the communication with @chrisgodfrey-deriv no need to have slack notification on Staging releases